priv/repo/seeds.exs

```diff
@@ -1,11 +1,4 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     CoursePlanner.Repo.insert!(%CoursePlanner.SomeModel{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+CoursePlanner.Repo.delete_all CoursePlanner.User
```

Comment:
I'm finding it difficult to separate into different files. I came up with a mix task, so it may be deemed refactoring, but according to [this issue](https://github.com/phoenixframework/phoenix_ecto/issues/8), this is a good middle-ground.